tests/ip_config_test.py

```diff
@@ -262,13 +264,39 @@ def test_intf_unknown_vrf_bind(self):
         print(result.exit_code, result.output)
         assert result.exit_code != 0
         assert result.output == INVALID_MGMT_VRF_MSG
-
+        
         result = runner.invoke(config.config.commands["vrf"].commands["add"], ["mgmt"], obj=obj)
         print(result.exit_code, result.output)
         result = runner.invoke(config.config.commands["interface"].commands["vrf"].commands["bind"], ["Ethernet64", "mgmt"], obj=obj)
         print(result.exit_code, result.output)
         assert result.exit_code == 0
 
+    @patch("config.validated_config_db_connector.ValidatedConfigDBConnector.validated_mod_entry", mock.Mock(side_effect=ValueError))
+    @patch("validated_config_db_connector.device_info.is_yang_config_validation_enabled", mock.Mock(return_value=True))
+    @patch("config.main.ConfigDBConnector.get_entry", mock.Mock(return_value={"mgmtVrfEnabled": "false"}))
+    def test_add_intf_unknown_vrf_bind_yang_validation(self):
```

Comment:
1. I have now removed `bind` from the testname, as the bind command is no longer tested in this test function
2. This returns non-zero exit code because I mocked set_entry and mod_entry to throw ValueError. This mock simulates an invalid ConfigDB possibly before the `config vrf add mgmt` command is able to go through due to some unexpected ConfigDB setting